### ISSUE TITLE
Updating Blog

### ISSUE THREAD

**Issue by Mzkhan25** [taniarascia/react-tutorial#6]
I've started learning React from blog, and while it's extremely well written, but i did face one issue. 

In **State** section, you used the following code

![image](https://user-images.githubusercontent.com/14289218/60448979-1d9a4400-9c40-11e9-9890-78c375f35f84.png)

where as, it should have been 

`  <div className="container">
      <Table characterData={this.state.characters} removeCharacter={this.removeCharacter} />
    </div>`

Otherwise, "characters" isn't recognized by our "Table" element. 

![image](https://user-images.githubusercontent.com/14289218/60449032-3c003f80-9c40-11e9-8980-be91c1134d05.png)

React might have been updated since then, due to which this error/issue is coming now. 

### Resolution

So, basically, the blog does mention that we need to add 
"`const { characters } = this.state`". This line kept skipping me eyes.

**Comment by Mzkhan25** [comment]
Okay, never mind. Apparently, i kept missing the following line from the blog. 

![image](https://user-images.githubusercontent.com/14289218/60449278-baf57800-9c40-11e9-8564-5568d8127b16.png)


Closing this issue but letting it remain here in-case someone else runs into it. Please feel free to delete it if you don't find it appropriate :)

**Comment by taniarascia** [comment]
You can update the post to include `const { characters } = this.state` above the code and make it more obvious. :)

**Comment by Mzkhan25** [comment]
Sure :)
